integration-tests/src/test/java/com/datastax/oss/driver/mapper/StatementAttributesIT.java

```diff
@@ -198,11 +243,35 @@ private void validateQueryOptions(QueryLog log) {
     @Insert
     void save(Simple simple, StatementAttributes attributes);
 
+    @Insert
+    @WithStatementAttributes(
+      consistencyLevel = "ANY",
+      serialConsistencyLevel = "QUORUM",
+      pageSize = 13
+    )
+    void save2(Simple simple);
+
     @Delete
     void delete(Simple simple, StatementAttributes attributes);
 
+    @Delete
+    @WithStatementAttributes(
+      consistencyLevel = "ANY",
+      serialConsistencyLevel = "QUORUM",
+      pageSize = 13
+    )
+    void delete2(Simple simple);
+
     @Select
     Simple findByPk(UUID pk, StatementAttributes attributes);
+
+    @Select
+    @WithStatementAttributes(
```

Comment:
Are there any edge cases that exclude each other in the `@WithStatementAttributes` configuration?
For example, do `serialConsistency` can be higher than `consistencyLevel`?
I am asking if we should add some kind of validation or we want to pass those properties to actual statements as they are

Reply:
No, the attributes are independent from each other.